ui/admin/tests/e2e/tests/auth-method.spec.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
```

Comment:
Can you check if we need to make these `BUSL` now?

Reply:
This repo is under MPL-2.0 at the moment